helm/aws-operator/templates/rbac.yaml

```diff
@@ -120,6 +120,14 @@ rules:
       - "/healthz"
     verbs:
       - get
+
+  # The aws-operator needs access for events to write those to the belonging objects.
+  - apiGroups:
+      - ""
+    resources:
+      - events
+    verbs:
+      - "*"
```

Comment:
Object creation does not mean object deletion, even though both are writing operations. Can we make this more specific?

Reply:
Yes I agreee I tested something with searching events and I wanted to change it back, but good catch 👍